src/main/java/Duke.java

```diff
@@ -1,10 +1,185 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+    private static final String[] INVALID_INPUT_MESSAGE = {
```

Comment:
I like how you adhere to using uppercase and underscore to separate words when naming constants